From fad492051ef070fd5ee72771a37cb116f230de5f Mon Sep 17 00:00:00 2001
From: Sean Busbey <busbey@cloudera.com>
Date: Sat, 31 Jan 2015 14:43:25 -0600
Subject: [PATCH 0014/1164] CLOUDERA-BUILD hbase handler should list
 dependency on high-scale-lib.

Ref: CDH-24808
(cherry picked from commit c3f847a58abc115c2fa1f28b4b14f7a7815ba41f)

Conflicts:
	pom.xml
---
 hbase-handler/pom.xml |    5 +++++
 pom.xml               |    1 +
 2 files changed, 6 insertions(+)

diff --git a/hbase-handler/pom.xml b/hbase-handler/pom.xml
index 6e6621d..cff828b 100644
--- a/hbase-handler/pom.xml
+++ b/hbase-handler/pom.xml
@@ -41,6 +41,11 @@
     </dependency>
     <!-- inter-project -->
     <dependency>
+      <groupId>com.github.stephenc.high-scale-lib</groupId>
+      <artifactId>high-scale-lib</artifactId>
+      <version>${high-scale-lib.version}</version>
+    </dependency>
+    <dependency>
       <groupId>commons-lang</groupId>
       <artifactId>commons-lang</artifactId>
       <version>${commons-lang.version}</version>
diff --git a/pom.xml b/pom.xml
index d81e436..36df44d 100644
--- a/pom.xml
+++ b/pom.xml
@@ -127,6 +127,7 @@
     <hadoop.bin.path>${basedir}/${hive.path.to.root}/testutils/hadoop</hadoop.bin.path>
     <hbase.hadoop1.version>${cdh.hbase.version}</hbase.hadoop1.version>
     <hbase.hadoop2.version>${cdh.hbase.version}</hbase.hadoop2.version>
+    <high-scale-lib.version>1.1.1</high-scale-lib.version>
     <!-- httpcomponents are not always in version sync -->
     <httpcomponents.client.version>${cdh.httpcomponents.version}</httpcomponents.client.version>
     <httpcomponents.core.version>${cdh.httpcomponents.core.version}</httpcomponents.core.version>
-- 
1.7.9.5

